metricbeat/module/prometheus/collector/_meta/data.json

```diff
@@ -1,25 +1,30 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
+    "agent": {
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
+    "event": {
+        "dataset": "prometheus.collector",
+        "duration": 115000,
+        "module": "prometheus"
+    },
     "metricset": {
-        "host": "prometheus:9090",
-        "module": "prometheus",
-        "name": "collector",
-        "namespace": "collector",
-        "rtt": 115
+        "name": "collector"
     },
     "prometheus": {
-        "collector": {
-            "label": {
-                "event": "add",
-                "role": "node"
-            },
-            "prometheus_sd_kubernetes_events_total": {
-                "value": 0
-            }
+        "http_request_duration_microseconds_count": 0,
```

Comment:
An alternative here would have been `prometheus.metrics.*` to not conflict with labels. Will also allow us to add more metadata in the future.